services/qs-spark-service/src/main/java/ai/quantumics/api/service/impl/RuleJobServiceImpl.java

```diff
@@ -107,6 +108,7 @@ public ResponseEntity<Object> runRuleJob(RuleJobRequest ruleJobRequest, int user
                     ruleJob.setRuleId(ruleId);
                     ruleJob.setJobStatus(RuleJobStatus.INPROCESS.getStatus());
                     ruleJob.setUserId(userId);
+                    ruleJob.setActive(true);
                     ruleJob.setCreatedDate(QsConstants.getCurrentUtcDate());
```

Comment:
ruleJob.setModifiedBy(controllerHelper.getFullName(userObj.getQsUserProfile())); - this can be avoided as createdBy use the same method to get fullname. ruleJob.setModifiedBy(ruleJob.getCreatedBy()) can be used